src/test_typing.py

```diff
@@ -1381,6 +1381,9 @@ def __anext__(self) -> T_a:
         exec(ASYNCIO_TESTS)
     except ImportError:
         ASYNCIO = False
+else:
+    # fake names for the sake of static analysis
```

Comment:
I would add ``asyncio = None`` here (as you do for PY36), just for completeness. But I don't want to delay the PR because of this minor thing.